### ISSUE TITLE
Serious issues 

### ISSUE THREAD

**Issue by Pankajrsjput0** [tyron12233/CodeAssist#690]
there is no option for import a file to project becuse of android 13 there is no access to data folder.

if can't import my custome icon to project what I can do then

**Comment by rabi347** [comment]
That is a good point. I am not sure why the default folder is set to the data folder. There should be an option to let us choose the folder.

**Comment by raccoonsteven** [comment]
Yeah, It's totally fucked up everything for me and probably most people.
